### DISCUSSION TITLE
How to export all folders / entire library and transfer to new mac

### DISCUSSION THREAD

**Discussion by kiamrehorces** [rsayfutdinov/Pikka#28] [Q&A] [answered]
Hi team, I am moving to a different mac. How do I carry over all color pallettes, i.e. my entire library?

Thank you!

**Comment by rsayfutdinov** [comment, marked as answer]
> Hi team, I am moving to a different mac. How do I carry over all color pallettes, i.e. my entire library?
> 
> 
> 
> Thank you!

Hi,

You could just turn on the iCloud sync in the settings.

Regards,
Ruslan

**Comment by kiamrehorces** [comment]
I overlooked this. Thanks Ruslan. Resolved :)

**Comment by maskinmedia** [comment]
Cant get iCloud sync to work. All tick boxes are ticked, but on the new computer (M2 MBAir) Pikka's colorlib remains empty. Can you tell me the location of the library so I can move it manually. Please.
